packages/charts/react-charts/library/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -1660,6 +1667,14 @@ export const transformPlotlyJsonToGVBCProps = (
     hideLegend,
     roundCorners: true,
     showYAxisLables: true,
+    roundedTicks: true,
+    ...(input.layout?.xaxis?.range
+      ? {
+          xMinValue: input.layout.xaxis.range[0],
```

Comment:
Create a function for this. Avoid code duplication. Refer ...getYMinMaxValues spread operation